core/src/main/scala/monocle/std/Validation.scala

```diff
@@ -0,0 +1,20 @@
+package monocle.std
+
+import monocle.{Prism}
+import scalaz.{Validation, Maybe, Success, Failure}
+import Maybe.{Just, Empty}
+
+object validation extends ValidationFunctions
+
+trait ValidationFunctions {
+
+  def success[E, A]: Prism[Validation[E, A], A] =
+    Prism[Validation[E, A], A](_.toMaybe)(Success.apply)
+
+  def failure[E, A]: Prism[Validation[E, A], E] =
+    Prism[Validation[E, A], E]{
+      case Success(a) => Empty()
```

Comment:
if you want, you can use fold instead of pattern matching :https://github.com/scalaz/scalaz/blob/series/7.2.x/core/src/main/scala/scalaz/Validation.scala#L58-L61